src/sgame/sg_cmds.cpp

```diff
@@ -4441,6 +4450,56 @@ void Cmd_PrivateMessage_f( gentity_t *ent )
 	}
 }
 
+
+/*
+=================
+Cmd_ReplyPrivateMessage_f
+
+Reply by private message to the last private message sender
+=================
+*/
+
+void Cmd_ReplyPrivateMessage_f(gentity_t *ent)
+{
+	char *msg;
+	int target;
+
+	if (!g_privateMessages.Get() && ent)
+	{
+		ADMP( "\"" N_("Sorry, but private messages have been disabled") "\"" );
+		return;
+	}
+
+	if (trap_Argc() < 2)
+	{
+	ADMP( QQ( N_("usage: /r [message]") ) );
+	return;
+	}
+
+	if ( level.time - ent->client->pers.lastPrivateMessageSenderTime < 3000 )
+	{
+		ADMP( "\"" N_("More than one possible recipient, refusing to send.") "\"" );
+		ADMP( va( "%s %s", QQ(N_("You would have replied to $1$")), g_entities[target].client->pers.netname));
+        return;
+	}
+
+	msg = ConcatArgs(1);
+	target = ent->client->pers.lastPrivateMessageSender;
+	if (target == -1 || !G_SayTo(ent, &g_entities[target], SAY_PRIVMSG, msg))
+    {
+        ADMP("\"" N_("No one to reply to.") "\"");
+        return;
+    }
+	if (G_SayTo(ent, &g_entities[target], SAY_PRIVMSG, msg))
```

Comment:
G_SayTo was already done inside the `if` expression 5 lines up so it doesn't need to be done again.

Also the preceding lines are indented with spaces. They should use tab to match the rest of the file

Reply:
it is now out of the "if" and, I replaced spaces by tabs